pkgs/applications/graphics/renderdoc/default.nix

```diff
@@ -17,8 +17,14 @@ stdenv.mkDerivation rec {
   buildInputs = [
     qtbase qtsvg xorg.libpthreadstubs xorg.libXdmcp qtx11extras vulkan-loader
   ];
+
   nativeBuildInputs = [ cmake makeWrapper pkgconfig python3 bison autoconf automake pcre ];
 
+  SWIG_PACKAGE = fetchurl {
+    url = "https://github.com/baldurk/swig/archive/renderdoc-modified-1.zip";
+    sha256 = "0xxvkkaa5rp2xnnms6i1i4kbvq9fj3is0kiawwdix4hfys16m4mz";
```

Comment:
Is this url stable?

Reply:
Yeah: https://github.com/baldurk/swig/releases/tag/renderdoc-modified-1

Reply:
These tarballs are generated by github. On regeneration they might change also the content does not change. Therefor it is required `fetchzip` or even better `fetchFromGitHub`, which is a wrapper around `fetchzip`.

Reply:
Okay, I already tried `fetchFromGitHub`, but gave up after some tries. I will try again, I think I have an idea was is wrong.

Reply:
@bkchr What errors did you encounter?